components/NavBar/navbar.module.css

```diff
@@ -32,51 +32,44 @@
   width: 4em;
   height: 4em;
 }
-.profilePic > img {
+.profilePic>img {
   border-radius: 50%;
   cursor: pointer;
 }
-.profilePic > img {
+.profilePic>img {
   width: 100%;
   height: 100%;
   border-radius: 50%;
   cursor: pointer;
```

Comment:
I think code from line no 35 through 38 is not needed. Repeated css.

Reply:
@sumitd94 Changes has been made please check